src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -213,6 +213,15 @@ def parse(self, backend, x509_obj):
                         "Critical extension {0} is not currently supported"
                         .format(oid), oid
                     )
+                else:
+                    # Dump the DER payload into an UnrecognizedExtension object
```

Comment:
This can't be implemented as a `handler` because `X509V3_EXT_d2i` returns `NULL` on rando oids I assume?